lib/llvm/compiler.cpp

```diff
@@ -851,10 +849,7 @@ class FunctionCompiler {
         Builder.positionAtEnd(
             LLVM::BasicBlock::create(LLContext, F.Fn, "ret_call_ref.end"));
         break;
-        // LEGACY-EH: remove the `Catch` cases after deprecating legacy EH.
-        // case OpCode::Catch:
-        // case OpCode::Catch_all:
-        // case OpCode::Try_table:
+      // case OpCode::Try_table:
```

Comment:
Should we remove this too?

Reply:
This is the EH proposal in AOT which is not implemented.